src/features/DisplayControls/NursingTasks/styles/TaskTile.scss

```diff
@@ -0,0 +1,65 @@
+.tile {
```

Comment:
can we change the class name? it is too generic and might have the same class name in Bahmni app